packages/pagination/src/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useIdAllocator } from '@leafygreen-ui/hooks';
+import ChevronLeft from '@leafygreen-ui/icon/dist/ChevronLeft';
+import ChevronRight from '@leafygreen-ui/icon/dist/ChevronRight';
+import IconButton from '@leafygreen-ui/icon-button';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
+import { Option, Select } from '@leafygreen-ui/select';
+import { Body } from '@leafygreen-ui/typography';
+
+import { baseStyles, flexSectionStyles } from './Pagination.styles';
+import { PaginationProps } from './Pagination.types';
+import {
+  getCurrentRangeString,
+  getRangeMaxString,
+  getTotalNumPages,
+} from './utils';
+
+const DEFAULT_ITEMS_PER_PAGE_OPTIONS = [10, 25, 50];
+
+function Pagination({
+  id: idProp,
+  className,
+  itemsPerPage = DEFAULT_ITEMS_PER_PAGE_OPTIONS[0],
+  itemsPerPageOptions = DEFAULT_ITEMS_PER_PAGE_OPTIONS,
+  onItemsPerPageOptionChange,
+  currentPage = 1,
```

Comment:
I assume you used Storybook controls for this? I didn't really check for this in the component because it's impossible to achieve with the controls we provide. This means this is only possible when the user passes in incorrect values as props.

If I'm not mistaken, there isn't a way to use prop-types to validate the value of a prop against the value of another prop even using a custom validator. I could probably `console.error` or throw an error in cases where the prop values are incorrect though. Let me add that